src/main/java/org/coursesjava/model/User.java

```diff
@@ -0,0 +1,17 @@
+package org.coursesjava.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class User {
+    private int id;
+    private String name;
+    private String nickname;
+    private String birthday;
+    private String password;
+    private Account account;
```

Comment:
it's redundant as you can get user's account by accountRepository.getByUser(userId);